test/puppetlabs/trapperkeeper/services/status/status_core_test.clj

```diff
@@ -124,10 +124,7 @@
           (is (every? #(or (< 0 %) (= -1 %)) (vals (:non-heap-memory jvm-metrics))))
           (is (= #{:max :used} (ks/keyset (:file-descriptors jvm-metrics))))
           (is (every? #(< 0 %) (vals (:file-descriptors jvm-metrics))))
-          (is (every? #(= #{:count :total-time-ms} (ks/keyset %))
+          (is (every? #(= #{:count :total-time-ms :last-gc-info} (ks/keyset %))
                       (vals (:gc-stats jvm-metrics))))
-          (is (every? #(<= 0 %)                             ;; Possible that no major collections occurred.
-                      (mapcat #(vals %)
-                              (vals (:gc-stats jvm-metrics)))))
```

Comment:
I dropped this test as Schema is doing a good job of enforcing assertions about the structure and data type of the response, re-writing the same assertions in a test is a lot of effort just to duplicate the same checks.